cmd/hypper/hypper.go

```diff
@@ -23,7 +24,11 @@ func main() {
 	logger := logcli.NewStandard()
 	log.Current = logger
 
-	actionConfig := new(helmAction.Configuration)
+	helmActionConfig := new(helmAction.Configuration)
+	actionConfig := &action.Configuration{
+		helmActionConfig,
+	}
```

Comment:
If you change the way the embedding is handled you leave vet unchanged.

```go
actionConfig := &action.Configuration{
	Configuration: helmActionConfig,
}
```

There's a blog post with a nice example at https://dave.cheney.net/2015/05/22/struct-composition-with-go

This idea applies in the test, too. Then you can strip out the `go vet` modifications.

Reply:
doh, thanks :).

Done. Force-pushed to amend commit messages and drop some commits too.